docs/android/inpath/implementationSteps.md

```diff
@@ -33,56 +33,108 @@ To implement the SDK's In Path flow within your app, please use the following st
 
 ---
 
-## Initialise the In Path flow with the SDK Builder <br/>
-Please make sure to set the following properties: 
-
-{: .note}
-.setEnvironment(CartrawlerSDK.Environment.PRODUCTION) must be set when submitting your app to the Play Store.
-
-```java
-CartrawlerSDK.Builder()
-           .setRentalInPathClientId(clientId(activity, palette))
-           .setEnvironment(environment)
-           .setCurrency(currency)
-           .setCountry(countryISO)
-           .setFlightNumberRequired(true)
-           .setPassenger(passenger)
-           .setAccountId("123")
-           .setLogging(true)
-           .setPickupTime(getPickUpDate())
-           .setPickupLocation("DUB")
-           .setDropOffLocationId(11)
-           .setDropOffTime(getDropOffDate())
-           .setTheme(getSelectedTheme(palette))
-           .startRentalInPath(activity, REQUEST_CODE_IN_PATH)
-           
-//Setup your lead passenger object
-val passenger = CartrawlerSDKPassenger(
-             firstName = "John",
-             lastName = "Smith",
-             email = "john@example.com",
-             phoneCountryCode = "353",
-             phoneNumber = "81234567",
-             address = "Dundrum Business Park",
-             city = "Dublin",
-             postcode = "D14 R7V2",
-             country = "IE", 
-             flightNumber = "EZY130",
-             age = "26", //Default age is 30
-             membershipId = "123456") 
+### Initialise the SDK <br/>
+
+First we need to initialise the SDK by calling the init method in the CartrawlerSDK class as follow:
+
+```kotlin
+val partnerImplementationID = "your-implementation-id-here"
+val environment = CTSdkEnvironment.DEVELOPMENT // CTSdkEnvironment.PRODUCTION
+
+CartrawlerSDK.init(partnerImplementationID, environment)
+```
+
+{: .important }
+The `implementationID` is needed by the SDK since it's used to fetch some configuration, it's good to call `CartrawlerSDK.init` in your application class;<br/>
+Don't forget to use `CTSdkEnvironment.PRODUCTION` when submitting your app to the Play Store;
```

Comment:
Maybe could just fix a bit the punctuation/wording to make it a bit clearer 
"The `implementationID` is needed by the SDK since it's used to fetch some configuration. It's recommended to call `CartrawlerSDK.init` in your application class.<br/>"